app/service/send_notification.py

```diff
@@ -37,6 +38,12 @@ def validate_created_by(service, created_by_id):
         raise BadRequestError(message=message)
 
 
+def create_one_off_reference(template_type):
+    if template_type != LETTER_TYPE:
```

Comment:
It doesn't make a difference, but I prefer to write/read if statement as positive.